tests/fuzz/fuzz-wire-wireaddr.c

```diff
@@ -0,0 +1,44 @@
+#include "config.h"
```

Comment:
Nit: this target is quite different from the other `fuzz-wire-wireaddr` targets, which all fuzz code in `wire/`.

Let's just call this `fuzz-wireaddr`.